next/yarn.lock

```diff
@@ -0,0 +1,2876 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
wait is this file meant to be here, there's already a package-lock.json (we use npm here?)